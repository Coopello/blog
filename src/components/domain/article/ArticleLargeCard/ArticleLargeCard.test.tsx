import renderer from "react-test-renderer";

import { ArticleLargeCard } from "./";

describe("domain/article/ArticleLargeCard", () => {
  it("Snap Shot", () => {
    const component = renderer.create(
      <ArticleLargeCard
        imageUrl={"https://placehold.jp/512x512.png"}
        description={
          "Firestore は〇〇です。\nガチ完成なのでぜひ読んでくださいね！"
        }
        color={"#00b8d2"}
        tags={["Front-end", "React"]}
      />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
