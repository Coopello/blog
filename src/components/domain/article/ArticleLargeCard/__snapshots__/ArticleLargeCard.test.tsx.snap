// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`domain/article/ArticleLargeCard Snap Shot 1`] = `
<div
  className="flex max-w-full flex-1 flex-col gap-3 overflow-x-hidden rounded bg-custom-white p-6"
>
  <div
    className="relative h-[250px] w-full max-w-[416px]"
  >
    <img
      alt="記事のサムネイル画像"
      className="object-contain"
      data-nimg="fill"
      decoding="async"
      loading="lazy"
      onError={[Function]}
      onLoad={[Function]}
      sizes="100vw"
      src="/_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=3840&q=75"
      srcSet="/_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=640&q=75 640w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=750&q=75 750w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=828&q=75 828w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=1080&q=75 1080w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=1200&q=75 1200w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=1920&q=75 1920w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=2048&q=75 2048w, /_next/image?url=https%3A%2F%2Fplacehold.jp%2F512x512.png&w=3840&q=75 3840w"
      style={
        {
          "bottom": 0,
          "color": "transparent",
          "height": "100%",
          "left": 0,
          "objectFit": undefined,
          "objectPosition": undefined,
          "position": "absolute",
          "right": 0,
          "top": 0,
          "width": "100%",
        }
      }
    />
  </div>
  <ul
    className="hidden-scrollbar flex w-full gap-2 overflow-y-clip overflow-x-scroll"
  >
    <li
      className="whitespace-nowrap"
    >
      <span
        className="rounded-sm py-1 text-sm"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          {
            "backgroundColor": "#E10718",
            "color": "white",
            "opacity": 1,
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        #
        Front-end
      </span>
    </li>
    <li
      className="whitespace-nowrap"
    >
      <span
        className="rounded-sm py-1 text-sm"
        onMouseEnter={[Function]}
        onMouseLeave={[Function]}
        style={
          {
            "backgroundColor": "#E10718",
            "color": "white",
            "opacity": 1,
            "paddingLeft": 12,
            "paddingRight": 12,
          }
        }
      >
        #
        React
      </span>
    </li>
  </ul>
  <div
    className="flex flex-1 flex-col gap-1"
  >
    <p
      className="text-base"
    >
      ニックネーム
    </p>
    <p
      className="text-sm text-custom-gray"
    >
      XXX Developer
    </p>
  </div>
</div>
`;
